api/valor_api/backend/metrics/detection.py

```diff
@@ -127,7 +133,22 @@ def _ap(
                 )
             )
 
-    return detection_metrics
+            recalls_across_thresholds.append(recalls[-1])
+
+        ar_metrics.append(
+            schemas.ARMetric(
```

Comment:
where is confidence coming in at computing AR?

Reply:
this is what the COCO max_dets does: it takes the N most confident predictions and considers those as positive predictions